### PR TITLE
Convert Petition comment to textarea

### PR DESCRIPTION
**Pull request by lionick**
`alter table cm_co_petition_attributes alter column value type varchar(4000) using value::varchar(4000);`

### CODE REVIEW

File: app/View/CoPetitions/petition-attributes.inc [hunk 1]
```diff
@@ -559,6 +559,10 @@
                 // Render the form field, label, and error (if any)
                 // error is output by default with class "error-message"
                 print $this->Form->label($fieldName, filter_var($ea['label'], FILTER_SANITIZE_SPECIAL_CHARS) . $requiredSymbol, array('class' => 'mdl-textfield__label'));
+                if( $ea["attribute"] == 'e:textfield' ) {
+                  $args['type'] = 'textarea';
+                  $args['maxlength'] = '4000';
+                }
                 print $this->Form->input($fieldName, $args);
```
Comment (ioigoume) [inline app/View/CoPetitions/petition-attributes.inc]: prefer to call `textarea` explicitly like this `$this->Form->textarea`

File: app/View/CoPetitions/petition-attributes.inc [hunk 2]
```diff
@@ -611,7 +615,21 @@
                 print $this->Time->nice($co_petition_attribute_values[ $ea['id'] ][ $ea['field'] ], $vv_tz);
                 break;
               default:
-                print $co_petition_attribute_values[ $ea['id'] ][ $ea['field'] ];
+                if($ea["attribute"] == 'e:textfield' && strlen($co_petition_attribute_values[ $ea['id'] ][ $ea['field'] ]) > 100) {
+                  $string = $co_petition_attribute_values[ $ea['id'] ][ $ea['field'] ];
```
Comment (ioigoume) [inline app/View/CoPetitions/petition-attributes.inc:621]: move string formatter to a method in `AppHelper`

**Review by ioigoume — CHANGES_REQUESTED** [review]

**Review by ioigoume — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**